src/Goteo/Console/Console.php

```diff
@@ -35,9 +35,9 @@ static public function get() {
 
 			self::$app = App::getService('console');
             self::$console = new Application();
-            /** @var EventDispatcherInterface $dispatcher */
-            $dispatcher = App::getService('dispatcher');
-            self::$console->setDispatcher($dispatcher);
+            /** @var EventDispatcherInterface $consoleDispatcher */
+            $consoleDispatcher = App::getService('console_dispatcher');
+            self::$console->setDispatcher($consoleDispatcher);
```

Comment:
@davidbeig here it was the clue about why the `ConsoleDispatcher` wasn't handling the `ConsoleListeners`

Reply:
this is because of this change 
![image](https://user-images.githubusercontent.com/1401520/140320112-16dd729a-a2d2-4060-ba0e-63959d2efe0a.png)

Reply:
Alright, however, I believe now is fine as well, isn't it? 

And perhaps it's even more clear which dispatcher in being used than with `self::$app->getDispatcher()`.